### PR TITLE
added session service

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -150,5 +151,5 @@ module.exports = function(options) {
 	return app;
 };
 
-module.exports.Router = express.Router
-
+module.exports.Router = express.Router;
+module.exports.services = serviceMatchers;
```
Comment (matthew-andrews) [inline main.js]: wai?

Comment (matthew-andrews) [inline main.js]: do you think it might be time for a separate `next-services` repo?

Comment (wheresrhys) [inline main.js]: we were talking about exposing the info so radiator can automatically build graphs for each service we're collecting for.

What else would go in next-services?

Comment (matthew-andrews) [inline main.js]: oh just thought you might want to have the array in a separate repo and then pull it in where you need it
but i'm fine with exposing it like this

**Pull request merged**